core/templates/dev/head/pages/exploration_editor/simple_editor_tab/ScrollSyncDirective.js

```diff
@@ -13,33 +13,30 @@
 // limitations under the License.
 
 /**
- * @fileoverview Directive used to mark parts of the editor that have a sidebar
+ * @fileoverview Directives used to mark parts of the editor that have a sidebar
```

Comment:
"Directive" -- we're going with a one-directive-per-file system. See comment below, too.